src/js/components/FormField/doc.js

```diff
@@ -175,6 +175,18 @@ export const themeDoc = {
     type: 'React.Element',
     defaultValue: undefined,
   },
+  'formField.error.size': {
+    description: `The size of the error messsage to be displayed.
+     The default size is medium.`,
+    type: 'string',
+    defaultValue: `
+      xsmall: '12px'
+      small: '14px'
+      medium: '18px'
+      large: '22px'
+      xlarge: '26px
+    `,
+  },
```

Comment:
```suggestion
  'formField.error.size': {
    description: `The size of the error messsage to be displayed.
     The default size is medium.`,
    type: 'string',
    defaultValue: medium',
  },
  'formField.error.size.small': {
    description: `The size of the error messsage to be displayed.
     The default size is medium.`,
    type: 'string',
    defaultValue: '12px',
  },
    'formField.error.size.medium': {
    description: `The size of the error messsage to be displayed.
     The default size is medium.`,
    type: 'string',
    defaultValue: '18px',
  },
  // same for the rest of the sizes.
```
It is a little confusing when we say the type of the prop is a  string, but showing an example of an object, the above is a cleaner way to suggest the changes.

Reply:
yes, this make more sense, shall I do that for the rest of it too ?